tests/integration/dryrun_mixin_docs_test.py

```diff
@@ -0,0 +1,608 @@
+byte "voted"
+app_local_get_ex
+bnz voted
+//read existing vote candidate
+txna ApplicationArgs 1
+app_global_get
+bnz increment_existing
+pop
+int 0
+increment_existing:
+int 1
++
+store 1
+txna ApplicationArgs 1
+load 1
+app_global_put
+int 0 //sender
+byte "voted"
+txna ApplicationArgs 1
+app_local_put
+int 1
+return
+voted:
+pop
+int 1
+return
+failed:
+int 0
+return
+"""
+        drr = self.dryrun_request(source, app=dict(app_idx=1))
+        self.assertReject(drr)
+        self.assertError(drr, "invalid ApplicationArgs index 0")
+
+        drr = self.dryrun_request(source, app=dict(app_idx=1, args=[b"vote"]))
+        self.assertReject(drr)
+        self.assertNoError(drr)
+
+        sender = "42NJMHTPFVPXVSDGA6JGKUV6TARV5UZTMPFIREMLXHETRKIVW34QFSDFRE"
+        creator = "DFPKC2SJP3OTFVJFMCD356YB7BOT4SJZTGWLIPPFEWL3ZABUFLTOY6ILYE"
+        creator_data = Account(
+            address=creator,
+            status="Offline",
+            created_apps=[
+                Application(
+                    id=1,
+                    params=ApplicationParams(
+                        global_state=[
+                            TealKeyValue(
+                                key=b64_encode_hack("VoteBegin"),
+                                value=TealValue(type=2, uint=1),
+                            ),
+                            TealKeyValue(
+                                key=b64_encode_hack("VoteEnd"),
+                                value=TealValue(type=2, uint=1000),
+                            ),
+                        ]
+                    ),
+                ),
+            ],
+        )
+
+        accounts = [creator_data]
+
+        drr = self.dryrun_request(
+            source,
+            app=dict(
+                app_idx=1,
+                args=[b"vote"],
+                round=3,
+                creator=creator,
+                accounts=accounts,
+            ),
+        )
+        self.assertReject(drr)
+        self.assertNoError(drr)
+
+        sender_data = Account(
+            address=sender,
+            status="Offline",
+            apps_local_state=[ApplicationLocalState(id=1)],
+        )
+
+        accounts = [creator_data, sender_data]
+        drr = self.dryrun_request(
+            source,
+            sender=sender,
+            app=dict(
+                app_idx=1,
+                creator=creator,
+                args=[b"vote"],
+                round=3,
+                accounts=accounts,
+            ),
+        )
+        self.assertError(drr, "invalid ApplicationArgs index 1")
+
+        accounts = [creator_data, sender_data]
+        drr = self.dryrun_request(
+            source,
+            sender=sender,
+            app=dict(
+                app_idx=1,
+                creator=creator,
+                args=[b"vote", "test"],
+                round=3,
+                accounts=accounts,
+            ),
+        )
+        self.assertPass(drr)
+        DryRunHelper.pprint(drr)
```

Comment:
@tzaffi Minor:  I didn't inspect closely though it seems `pprint` results in _a lot_ of stdout during test runs.  Leaving the note to sanity check if it's straight-forward to reduce output (e.g. via log levels) at least when tests pass?